pkg/util/netsh/netsh.go

```diff
@@ -154,6 +161,16 @@ func (runner *runner) DeleteIPAddress(args []string) error {
 	return fmt.Errorf("error deleting ipv4 address: %v: %s", err, out)
 }
 
+// GetInterfaceToAddIP returns the interface name where Service IP needs to be added
+// IP Address needs to be added for netsh portproxy to redirect traffic
+// Reads Environment variable INTERFACE_TO_ADD_SERVICE_IP, if it is not defined then "vEthernet (HNSTransparent)" is returned
+func (runner *runner) GetInterfaceToAddIP() string {
```

Comment:
Is there a reason you are using an environment variable for this?  Generally this would be a flag to the proxy (we don't use environment variables except in rare conditions).